src/components/OptionsList.js

```diff
@@ -190,7 +190,10 @@ class OptionsList extends Component {
 
     render() {
         return (
-            <View style={[styles.flex1]}>
+
+            // need to set a height (0 works in this case) so that the view will scroll on mobile
+            // NOTE: the view will still fill its container since it has flex: 1 on it
+            <View style={[styles.flex1, {height: 0}]}>
```

Comment:
Heads up, inline styles are not permitted this should probably be moved to [`sizing.js`](https://github.com/Expensify/Expensify.cash/blob/8eaa4633ce743fe68c710c0aae4595dc2daa7089/src/styles/utilities/sizing.js#L6).

Why does this work exactly? 🤔